fluent-resmgr/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "fluent-resmgr"
 description = """
 Resource manager for Fluent localization resources.
 """
-version = "0.0.4"
+version = "0.0.5"
```

Comment:
Is there a reason not to use `0.1.0`? As is, there is no scope for differentiating major/minor/patch changes from each other.

Reply:
Versions in Rust are meant to be conveying the subjective state of maturity of the package. 

`x.y` indicates production ready. `0.x.y` indicates `ready for use`, `0.0.x` indicates `POC`. `fluent-resmgr` is POC.